src/components/modal/modal.component.tsx

```diff
@@ -16,6 +16,8 @@ export interface ModalContextProps {
 export const ModalContext = React.createContext<ModalContextProps>({});
 
 export interface ModalProps {
+  /** Custom class name  */
+  className?: string;
```

Comment:
is there a reason we need this prop here?

Reply:
Yes, because of this line of code
https://github.com/Sage/carbon/pull/5424/files#diff-bf208d8f6f4f6e642c90ddd1dc62829746a7b79b0b0eb934bd6ae7700612cdbeR161